pages/about.tsx

```diff
@@ -166,70 +165,23 @@ export default function AboutPage() {
         </Box>
       </SectionContainer>
 
-      <SectionContainer
-        id="our-partners"
-        sx={{
-          margin: 'auto', mb: 4, px: 3, maxWidth: '65rem',
-        }}
-        maxWidth={false}
-      >
-        <Typography variant="h2" sx={{ my: '3rem' }}>{content.ourPartners.header}</Typography>
-        <PaperSection sx={{ pt: 0, pb: 1 }}>
-          <Grid container>
-            <Grid item xs={12}>
-              <Box display="flex" justifyContent="center" sx={{ pt: 4 }}>
-                {content.ourPartners.ctaLink && (
-                  <Link href={content.ourPartners.ctaLink}>
-                    <ImageContainer
-                      alt=""
-                      src={content.ourPartners.imgsrc as string}
-                      width={442}
-                      height={248}
-                      style={{
-                        height: '124px',
-                        width: '221px',
-                        objectFit: 'contain',
-                      }}
-                    />
-                  </Link>
-                )}
-              </Box>
-            </Grid>
-            <Grid
-              item
-              xs={12}
-              sx={{
-                display: 'flex', flexDirection: 'column', pt: '0.5rem',
-              }}
-            >
-              {content.ourPartners.ctaLink && (
-                <Typography
-                  sx={{
-                    color: 'primary.dark',
-                    textAlign: 'center',
-                    my: 2,
-                  }}
-                  variant="subtitle1"
-                >
-                  <Link href={content.ourPartners.ctaLink} style={{ textDecoration: 'none' }}>
-                    {content.ourPartners.body}
-                  </Link>
-                </Typography>
-              )}
-              <Typography
-                sx={{
-                  textAlign: 'center',
-                  mt: 1,
-                  mb: 3,
-                  mx: 2,
-                }}
-                variant="body2"
-              >
-                {content.ourPartners.subheading2}
-              </Typography>
-            </Grid>
-          </Grid>
-        </PaperSection>
+      <SectionContainer key={content.partnersFrame[0].id} sx={{ margin: 'auto', px: 3, maxWidth: '65rem' }} maxWidth={false}>
+        <Typography variant="h2" sx={{ mt: '3rem' }}>{content.partnersFrame[0].title}</Typography>
+        <Grid container>
+          <iframe
+            id={`${content.partnersFrame[0].id}-partnersFrame`}
```

Comment:
```suggestion
            id={`${content.partnersFrame.id}-partnersFrame`}
```
by removing `[ ]`, we do not need this `[0]` anymore

Reply:
implemented